src/views/rocket.py

```
@@ -1,41 +1,61 @@
+    check_parachute_trigger: Optional[Any] = None
+    com_to_cdm_function: Optional[Any] = None
+    cp_position: Optional[Any] = None
+    motor_center_of_mass_position: Optional[Any] = None
+    nozzle_gyration_tensor: Optional[Any] = None
+    reduced_mass: Optional[Any] = None
+    stability_margin: Optional[Any] = None
+    static_margin: Optional[Any] = None
+    thrust_to_weight: Optional[Any] = None
+    total_lift_coeff_der: Optional[Any] = None
+
+    def __init__(self, **data):
```

Pylint (3.3.7)

[warning] Useless parent or super() delegation in method '__init__'

(W0246)